dashboard/templates/dashboard/index.html

```diff
@@ -5,8 +5,10 @@
     <!-- Load bootstrap css, jquery, bootstrap js -->
     <!-- common across all applications inheriting from dashboard -->
     <link href="{% static 'css/bootstrap.min.css' %}" rel="stylesheet">
-    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
-    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.1.0/js/bootstrap.min.js"></script>
+    <script src="{% static 'js/jquery-3.1.1.slim.min.js' %}"></script>
+    <script src="{% static 'js/popper.js' %}"></script>
+{#    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.1.0/js/bootstrap.min.js"></script>#}
```

Comment:
Since the functionality is tested with a local min.js file, we can have this comment removed.

Reply:
I just kept that for reference. I could remove the comment.